docs/hub/mlx-image.md

```diff
@@ -0,0 +1,89 @@
+# Using mlxim at Hugging Face
+
+[`mlxim`](https://github.com/riccardomusmeci/mlx-image) is an image models library built on Apple [MLX](https://github.com/ml-explore/mlx). It tries to replicate the great [timm](https://github.com/huggingface/pytorch-image-models) library from Ross Wightman, but for MLX models.
+
+
+## Exploring MLX on the Hub
+
+You can find `mlxim` models by filtering at the left of the [models page](https://huggingface.co/models?library=mlxim&sort=trending).
```

Comment:
```suggestion
You can find `mlxim` models by filtering using the `mlxim` library name, like in [this query](https://huggingface.co/models?library=mlxim&sort=trending).
```

I believe the library name won't appear in the list of available libraries because it's being added with `filter=false`. We can revisit when the library gets more traction :)

Reply:
Do you suggest keeping the sentence then?

Reply:
Yes, for sure, we can keep the filter URL, just not mention the left column in the models page!